web/backend/src/controllers/dela.ts

```diff
@@ -257,9 +253,11 @@ delaRouter.use('/*', (req, res) => {
     // translate UserIDs to another random ID.
     // bodyData.UserID = req.session.userId.toString();
 
-    // DEBUG: this is only for debugging and needs to be replaced before production
-    console.warn('DEV CODE - randomizing the SCIPER ID to allow for unlimited votes');
-    bodyData.UserID = makeid(10);
+    if (process.env.REACT_APP_RANDOMIZE_VOTE_ID === 'true') {
```

Comment:
doesn't this meen that bodyData.UserID will be empty if `REACT_APP_RANDOMIZE_VOTE_ID` is not true? if we introduce this switch, shouldn't we then decomment the line above and use it in case `REACT_APP_RANDOMIZE_VOTE_ID` is false?